roles/operator_build/tasks/build.yml

```diff
@@ -118,63 +118,63 @@
       }}}, recursive=True)}}
 
 - name: "{{ operator.name }} - Call manifests"  # noqa: name[template]
-  ci_make:
+  ci_script:
     dry_run: "{{ cifmw_operator_build_dryrun|bool }}"
     chdir: "{{ operator.src }}"
     output_dir: "{{ cifmw_operator_build_basedir }}/artifacts"
-    target: manifests
+    script: manifests
 
 - name: "{{ operator.name }} - Call docker-build"  # noqa: name[template]
-  ci_make:
+  ci_script:
     dry_run: "{{ cifmw_operator_build_dryrun|bool }}"
     chdir: "{{ operator.src }}"
     output_dir: "{{ cifmw_operator_build_basedir }}/artifacts"
-    target: docker-build
-    params:
-      IMG: "{{ operator_img }}"
+    script: make docker-build
+  environment:
+    IMG: "{{ operator_img }}"
```

Comment:
maybe we can pass that inline like we'd do with an actual call?
something like `make docker-build IMG={{ operator_img }}` ?

Reply:
+1

Reply:
ok. i am adding a params to the module options. we can call it anything else if you prefer please comment when i push the update. I chose params because then it makes the replacement easier. extra_args also works? extra_vars ? ;)

i will set those before the invocation of script.

Reply:
see changes in ci_script.py

 https://github.com/openstack-k8s-operators/ci-framework/pull/653/files#diff-3e84390170fc913d734714c261dad818ea4ef5f31f66ec69b2f65cd6795a846c


example output from the tests:

```
]$ cat ci_script_001_run_ci_make_with_a_param.sh 
#!/bin/bash
set -euo pipefail

exec > >(tee -i /tmp/logs/ci_script_001_run_ci_make_with_a_param.log) 2>&1
pushd /tmp/project_makefile
FOO_BAR=starwars  make help
popd

```

Reply:
I like it! Let's drop the "environment" then, and switch to "params".
It will make things far, far easier to parse in the action plugin, parsing the "environment" is a nightmare due to the multiple layers ansible is consuming.

Reply:
OK. As discussed in yesterday's reviews call, we will keep this new params addition to ci_script, and I'll go through and change all the ci_make to ci_script. When those ci_make were setting environment: instead I will move those to params: